vm/vm.go

```diff
@@ -213,8 +222,12 @@ func (vm *VM) Initialize(
 		return err
 	}
 
-	// Setup worker cluster
-	vm.workers = workers.NewParallel(vm.config.GetParallelism(), 100)
+	// Setup worker cluster for verifying signatures
+	//
+	// If [parallelism] is odd, we assign the extra
+	// core to signature verification.
+	sigParallelism := parallelism - rootGenParallelism
```

Comment:
`rootGenParallelism` can be 0 here... is that expected?

Reply:
no

Reply:
have a pending commit that cleans this up